Cargo.lock

```diff
@@ -1337,18 +1337,18 @@ checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
 [[package]]
 name = "openssl-src"
-version = "300.2.3+3.2.1"
+version = "111.28.1+1.1.1w"
```

Comment:
If we're going to do this, should get rid of the perl IPC-CMD thing for now?

Reply:
I didn't notice the downgrade `openssl-src` until now... I'd say we are at a very weird state currently. This fix is required for OpenSSL v3 which we could update to in the future, but it's just half-done: it's fixed with x64, but not fixed with x86 until now. I'd say within this downgrade commit I'll remove the perl IPC-CMD thing for both, but at the same time let's keep the previous commit that fixes the build for v3 on x86.